.github/workflows/test.yml

```diff
@@ -3,7 +3,7 @@ name: Tests
 on:
```

Comment:
I have noticed codecov will count test files towards the total percentage of code coverage, which seems wrong. Is there some way to fix that? https://app.codecov.io/gh/IMAP-Science-Operations-Center/imap_processing?search=&displayType=list (you can see `codice/tests/test_bin_data.py` has a warning)

Reply:
That is actually the one place we want 100% code coverage IMO! Otherwise, we have tests that are never run is what that is telling us.

The reason for this is because there is an `__main__` block that doesn't get hit when not run from the command line. We should remove that / change it over to an actual test function.
https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/b0d52a544ed05178cf5062045c1951614e0e7066/imap_processing/codice/tests/test_bin_data.py#L42